components/SupportSection/SupportSection.tsx

```diff
@@ -28,22 +34,18 @@ export function SupportSection() {
           </Subtitle>
         </TextWrapper>
         <LinksWrapper>
-          <ButtonGroup>
-            <div>
-              <ButtonLink href="https://jobs.lever.co/umaproject" target="_blank" rel="noreferrer">
-                <UpRightArrowLg />
-              </ButtonLink>
-              <ButtonText>Careers</ButtonText>
-            </div>
-          </ButtonGroup>
-          <ButtonGroup>
-            <div>
-              <ButtonLink href="https://docs.umaproject.org/" target="_blank" rel="noreferrer">
-                <UpRightArrowLg />
-              </ButtonLink>
-              <ButtonText>About</ButtonText>
-            </div>
-          </ButtonGroup>
+          <AnimatedLink href="https://jobs.lever.co/umaproject" target="_blank">
```

Comment:
had to rebuild this component to make it behave correctly. the original implementation made only the arrow icon clickable and not the text